.travis.yml

```diff
@@ -7,8 +7,9 @@ python:
 before_install:
   - sudo add-apt-repository ppa:ondrej/autotools -y
   - sudo add-apt-repository ppa:debian-med/ppa -y
+  - sudo add-apt-repository ppa:dluxen/boost -y
```

Comment:
this reminds me, we should switch Travis to Xenial, seeing that 16.04 is finally supported now